docs/src/development/transfer-dedicated.md

```diff
@@ -29,7 +29,7 @@ First, login to the cluster and run the following command:
 echo $PLATFORM_RELATIONSHIPS | base64 --decode | json_pp
 ```
 
-Which should give a JSON output containing something like this:
+Which gives a JSON output similar to:
```

Comment:
:+1: / :hammer_and_wrench: I think this is a good improvement, but if you're going to do it here, you should also do it for the exact same phrase below (112)